From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 19 Mar 2016 11:18:22 +0100
Subject: tests: add file extension for host mounts

otherwise we fail in gvir_sandbox_util_guess_image_format with

  "Cannot identify file extension in '/tmp/home'
---
 libvirt-sandbox/tests/test-config.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libvirt-sandbox/tests/test-config.c b/libvirt-sandbox/tests/test-config.c
index ac10bab..71a65c8 100644
--- a/libvirt-sandbox/tests/test-config.c
+++ b/libvirt-sandbox/tests/test-config.c
@@ -53,7 +53,7 @@ int main(int argc, char **argv)
     int ret = EXIT_FAILURE;
     const gchar *mounts[] = {
         "host-bind:/var/run/hell=/tmp/home",
-        "host-image:/etc=/tmp/home",
+        "host-image:/etc=/tmp/home/img.qcow2",
         "host-image:/etc=/tmp/home,format=qcow2",
         "host-bind:/tmp=",
         NULL
